Reword text files (#3)
bugsnag/bugsnag-expo: README.md

# Bugsnag error monitoring & reporting for Expo

Automatically detect JavaScript errors in Expo. Get cross-platform error detection for handled and unhandled errors with real-time error alerts and detailed diagnostic reports.

---

> This is a monorepo (managed with [Lerna](https://lerna.js.org/) containing Expo client [`@bugsnag/expo`](/packages/expo), along with plugins for internal functionality. See [packages](/packages) for a full list of contents.

Reviewer: ```suggestion
This is a monorepo (managed with [Lerna](https://lerna.js.org/) containing our Expo client [`@bugsnag/expo`](./packages/expo), our Expo helper CLI [`bugsnag-expo-cli `](./packages/expo-cli) and dependent plugins. See [packages](./packages) for a full list of contents.
```

> This is a monorepo (managed with [Lerna](https://lerna.js.org/) containing Expo client [`@bugsnag/expo`](/packages/expo), along with plugins for internal functionality. See [packages](/packages) for a full list of contents.

Reviewer: Do we really want to call this a "monorepo"?  As I understand it the term started with e.g. Facebook and Google having a single repo for all their code.  

`bugsnag-js` I can buy into because it has multiple "end result" packages (browser / node / RN). But this is very clearly focussed around Expo alone.

Perhaps the term has a new meaning in JS-world, which roughly means "a repo managed by Lerna"?  But Lerna itself says it is for "multi-package repositories", which seems a better term.

Not a hill I need to die on but just wanted to ask the question because it grates with me a little.

Author: I don't think monorepo means literally _all_ of your code has to be there, which matches the definition on wikipedia:

> In version control systems, a monorepo ... is a software development strategy where code for many projects is stored in the same repository

Reviewer: You wouldn't call "Bugsnag's Expo capabilities" a single project?
